### PR TITLE
trivial/refactoring: simple bls refactoring / adjustments

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

**Review by xdustinface — APPROVED** [review]
utACK

File: src/bls/bls_worker.cpp [hunk 1]
```diff
@@ -7,6 +7,8 @@
 #include <serialize.h>
 
 #include <util.h>
+#include <memory>
```
Comment (UdjinM6) [inline src/bls/bls_worker.cpp:10]: nit:
```suggestion

#include <memory>
```

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by xdustinface — APPROVED** [review]
utACK

**Pull request merged**